### ISSUE TITLE
Add rewind option to video (for instruction video)

### ISSUE THREAD

**Issue by marccolomer** [lookit/lookit-api#536]
**TL;DR**: I would like to have the rewind option for videos.

**Narrative**
We prefer to explain the instructions of the experiments using a video. However, if parents miss some information, they have either to re-watch all the video, or they will miss the information. The option of pause and rewind would be very useful for those parents who have missed some information.

**Acceptance Criteria**
If it is possible, this option would be considered close when it is implemented and parents can successfully rewind the video.

**Implementation Notes**
I don't know how difficult is to add a rewind option. One option that sounds easier is to go backwards frame by frame... it is slow, but still it gives you the option to go back.

**Comment by kimberscott** [comment]
Which frame type(s) are you currently using to show instruction videos? 

This isn't technically difficult - probably just a parameter `showControls` or similar to show the controls on the video element (which will give you pause/play as well as the little dot you can move to scan/rewind). For the exp-lookit-stimuli-preview and exp-lookit-instructions frames the controls are already shown. For exp-lookit-video we might want to also allow the frame not to proceed automatically (using a next button instead), like exp-lookit-images-audio - but at that point we might actually just want a separate frame type intended more for participant-controlled video to avoid a ton of interacting options for when it's possible to move on.

**Comment by marccolomer** [comment]
Hi Kim,

We were using "exp-lookit-video", but we switched to "exp-lookit-instructions" and it seems that we got what we wanted. So the option I was asking for it was already implemented. Thank you and sorry for the inconvenience!

Marc

**Comment by kimberscott** [comment]
Okay! If that turns out not to be a good solution for formatting reasons (e.g. you'd rather have the instructions video take up most of the screen) just let me know :)

**Comment by kimberscott** [comment]
just fyi there's also now the [exp-lookit-instruction-video](https://lookit.readthedocs.io/projects/frameplayer/en/latest/components/exp-lookit-instruction-video/doc.html) frame!
